charts/camunda-platform-alpha-8.8/templates/connectors/configmap.yaml

```diff
@@ -36,23 +36,30 @@ data:
 
     camunda:
       client:
-        zeebe:
-          {{-  $proto := (lower .Values.core.readinessProbe.scheme) -}}
-          {{- $baseURLInternal := printf "%s://%s" $proto (include "core.fullname" . | trimAll "\"") }}
-          rest-address: {{ printf "%s:%v%s" $baseURLInternal .Values.core.service.httpPort .Values.core.contextPath }}
-          grpc-address: {{ printf "%s:%v" $baseURLInternal .Values.core.service.grpcPort }}
+        {{-  $proto := (lower .Values.core.readinessProbe.scheme) -}}
+        {{- $baseURLInternal := printf "%s://%s" $proto (include "core.fullname" . | trimAll "\"") }}
+        enabled: true
+        rest-address: {{ printf "%s:%v%s" $baseURLInternal .Values.core.service.httpPort .Values.core.contextPath }}
+        grpc-address: {{ printf "%s:%v" $baseURLInternal .Values.core.service.grpcPort }}
+        worker:
           defaults:
             worker-threads: 10
             max-jobs-active: 32
-            stream-enabled: true
-        identity:
-          base-url: {{ include "camundaPlatform.identityURL" . | quote }}
-          audience: {{ include "core.authAudience" . | quote }}
+            stream-enabled: true          
+        {{- if eq .Values.global.security.authentication.method "basic" }}
+        mode: basic
+        auth:
+          {{- $user := first .Values.global.security.initialization.users }}
+          username: {{ $user.username | quote }}
+          password: {{ $user.password | quote }}
+        {{- else }}
```

Comment:
🔧 Does it make sense to just pick the first configured user? I think we should default to the first configured user, but grant users the ability to override this in their own values. I could imagine customers wanting to have a specific connectors users with limited permissions.

Reply:
it's indeed very short-sighted 😬 ideally we would have a dedicated connectors user preconfigured, customers may override that one fully.

However, that requires us to allow configuring authorizations as well, which is currently not possible looking at [InitializationConfiguration](https://github.com/camunda/camunda/blob/main/security/security-core/src/main/java/io/camunda/security/configuration/InitializationConfiguration.java), right? I would create a follow-up for that then.